docs/modules/ROOT/pages/common.adoc

```diff
@@ -0,0 +1,99 @@
+  given when the function is called.
+bug:: The _bug_ function is implemented in _ebcl/common/__init__.py_.
+  It prints an info text how to report a bug ticket if a tool hits an exception.
+promo:: The _promo_ function is implemented in _ebcl/common/__init__.py_.
+  It prints some advertising if the build worked well.
+log_exception:: The _log_exception_ annotation is implemented in _ebcl/common/__init__.py_.
+  It takes care of the default exception handling for all build tools.
+get_cache_folder:: The _get_cache_folder_ function is implemented in _ebcl/common/__init__.py_.
+  It decides which folder to use to cache packages and build results.
+filter_packages:: The _filter_packages_ function is implemented in _ebcl/common/deb.py_.
+  It implements package filtering according to a given _Version_ and _VersionRelation_.
+vds_to_package:: The _vds_to_package_ function is implemented in _ebcl/common/dependency.py_.
+  It converts a list of _VersionDepends_ to a list of _Package_s.
+sub_output_folder:: The _sub_output_folder_ function is implemented in _ebcl/common/files.py_.
+  It replaces the _$$RESULTS$$_ placeholder with the real output path.
+parse_scripts:: The _parse_scripts_ function is implemented in _ebcl/common/files.py_.
+  It parses a _scripts_ parameter and sets the default for all not given options.
+parse_files:: The _parse_files_ function is implemented in _ebcl/common/files.py_.
+  It parses a _host_files_ parameter, sets the default for all not given options, and takes care
+  of completing relative paths with respect to the config file containing the relative path.
+resolve_files:: The _resolve_files_ function is implemented in _ebcl/common/files.py_.
+  It implements the completion of relative paths with respect to the config file containing
+  the relative path.
+render_template:: The _render_template_ function is implemented in _ebcl/common/templates.py_.
+  It renders a Jinja2 template.
+parse_depends:: The _parse_depends_ function is implemented in _ebcl/common/version.py_.
+  It parses the dependency information given in Debian package metadata as _VersionDepends_.
+parse_package_config:: The _parse_package_config_ function is implemented in _ebcl/common/version.py_.
+  It parses a package given in a specification file as _VersionDepends_.
+parse_package:: The _parse_package_ function is implemented in _ebcl/common/version.py_.
+  It parses a single package given in a specification file as _VersionDepends_.
+
+== Types
+
+The module _ebcl/common/types_ provides the commonly used type enums
+and classes.
+
+CpuArch:: The _CpuArch_ enum is implemented in _ebcl/common/types/cpu_arch.py_.
+  It provides values for the CPU architecture of an artifact and is e.g. used
+  for tagging the Debian package objects.
+
+EnvironmentType:: The _EnvironmentType_ enum is implemented in
+  _ebcl/common/types/environment_type.py_ and used for tagging scripts
+  with the required execution environment.
```

Comment:
I don't like this at all. This is redundant and will be outdated fast.
Something like that should be generated automatically from the pydoc in the code, but never created manually